backend/controller/controller.js

```diff
@@ -189,6 +189,21 @@ export const createEvent = async (req, res) => {
   }
 };
 
+//Deletes an EventInstance
+export const deleteEventInstance = async (req, res) => {
+  try{
+    const {idofevent} = req.params;
+    const numdeleted = await EventInstance.destroy({where:{event_id: idofevent}});
+    if (numdeleted === 0){
+      return res.status(404).json({ message: "Event deleted successfully" });
+    }
+    return res.status(404).json({ message: "Failed to delete event" });
+  } catch (error){
+    console.error("Error creating event:", error);
```

Comment:
This callback function looks good. Logic looks correct.
Just a small fix: update the console.error to say "error deleting event"

Reply:
Logics are good. Change as @joshua041904 say and we will merge it.

Reply:
Cool!